### PR TITLE
Add task to create a bunch of notifications

### PR DESCRIPTION
**Pull request by saraycp**
Add notifications of different types to the database.

Run this task with `rails "dev:notifications:data[3]"` replacing 3 with any number to indicate how many times you want this code to be executed.

### CODE REVIEW

File: src/api/lib/tasks/dev.rake [hunk 1]
```diff
@@ -170,6 +170,57 @@ namespace :dev do
...
+
+      unless home_admin
+        home_admin = create(:project, name: admin.home_project_name)
+        create(:relationship, project: home_admin, user: admin, role: Role.hashed['maintainer'])
+      end
+
+      branched_project = Project.where(name: requestor.branch_project_name('home:Admin')).first ||
+                         create(:project, name: requestor.branch_project_name('home:Admin'))
+      admin_package = Package.where(name: 'admin_package').first ||
+                      create(:package_with_file, name: 'admin_package', project: home_admin, file_content: 'from admin home')
```
Comment (krauselukas) [inline src/api/lib/tasks/dev.rake:197]: @saraycp how about adding a timestamp or a random number to the package name? this way we could run the task a few times if a few more fake notifications are needed, without having always the same package name :)

Comment (saraycp) [reply src/api/lib/tasks/dev.rake:197]: Good idea!

Comment (saraycp) [reply src/api/lib/tasks/dev.rake]: @krauselukas I folloed your suggestion and added different names. But also improved to make it easier to generate many notification at a time.

Run this task with `rails "dev:notifications:data[3]"` replacing 3 with any number to indicate how many times you want this code to be executed.

File: src/api/lib/tasks/dev.rake [hunk 2]
```diff
@@ -170,6 +170,60 @@ namespace :dev do
...
+      unless home_admin
+        home_admin = create(:project, name: admin.home_project_name)
+        create(:relationship, project: home_admin, user: admin, role: Role.hashed['maintainer'])
+      end
+      requestor = User.where(login: 'Requestor').first || create(:confirmed_user, login: 'Requestor')
+      requestor_project = create(:project, name: "requestor_project_#{Time.now.to_i}")
+
+      repetitions.times do |repetition|
+        timestamp = Time.now.to_i
+        admin_package = create(:package_with_file, name: "admin_package_#{timestamp}_#{repetition}", project: home_admin, file_content: 'from admin home')
```
Comment (hennevogel) [inline src/api/lib/tasks/dev.rake:200]: Why not use faker? like `Faker::Lorem.word`

Comment (saraycp) [reply src/api/lib/tasks/dev.rake]: Much better, updated!

File: src/api/lib/tasks/dev.rake [hunk 3]
```diff
@@ -170,6 +170,59 @@ namespace :dev do
...
+          :bs_request_with_submit_action,
+          creator: requestor,
+          target_project: home_admin,
+          target_package: admin_package,
+          source_project: requestor_project,
+          source_package: requestor_package
+        )
+
+        # Will create a notification (ReviewWanted event) for this review.
+        User.session = requestor
```
Comment (eduardoj) [inline src/api/lib/tasks/dev.rake:213]: This assignment could be moved outside the loop, couldn't it?

Comment (saraycp) [reply src/api/lib/tasks/dev.rake:213]: sure thing

Comment (saraycp) [reply src/api/lib/tasks/dev.rake]: @eduardoj  done!

**Pull request merged**